package.json

```diff
@@ -10,6 +10,7 @@
     "web": "expo start --web",
     "build:android": "eas build --profile development --platform android",
     "build:ios": "eas build --profile development-ios --platform ios",
+    "generate:api": "x(){ openapi-generator-cli generate -i $1 -g typescript-axios -o src/api/generated --additional-properties=supportsES6=true,withInterfaces=true; };x",
```

Comment:
The `x()` stuff here is because of npm fuckiness where it didn't want to let me pass in a param for the path with $1. This is a clean-ish workaround.